src/app/shared/components/stars-rate/stars-rate.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-stars-rate',
+  templateUrl: './stars-rate.component.html',
+  styleUrls: ['./stars-rate.component.scss']
+})
+export class StarsRateComponent {
+  @Input() rating!: number;
+  starsArray!: string[];
+
+  ngOnInit() {
+    this.starsArray = this.createStarsArray(this.rating);
+  }
+
+  createStarsArray(rating: number) {
+    let ratingVal = Math.min(Math.max(rating, 0), 5);
```

Comment:
why here let and not const?
use linter setting to be easier handle such mistakes

Reply:
okay. Thank you